pkg/plugins/ifaces.go

```diff
@@ -44,6 +44,10 @@ type Manager interface {
 	GetPluginDashboards(orgID int64, pluginID string) ([]*PluginDashboardInfoDTO, error)
 	// GetPluginMarkdown gets markdown for a certain plugin/name.
 	GetPluginMarkdown(pluginID string, name string) ([]byte, error)
+	// CacheDashboardUIDBySlug is used to map plugin dashboard slugs into UIDs without making additional database queries.
+	CacheDashboardUIDBySlug(orgID int64, slug string, uid string)
```

Comment:
This method feels a bit weird on the PluginManager interface and/or exposing this functionality for consumers outside the PluginManager is not optimal.